src/Shared/Recruit.Shared.Web/Helpers/PagingHelper.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace Esfa.Recruit.Shared.Web.Helpers
+{
+    public static class PagingHelper
+    {
+        public static int GetPageNoOfSelectedItem(int totalNumberOfPages, int maxResultsPerPage, int nonZeroIndexOfSelectedItem)
+        {
+            int page;
+            var pageNoOfSelectedItem = (int)Math.Floor((nonZeroIndexOfSelectedItem - 1) / (double)maxResultsPerPage) + 1;
+
+            if (pageNoOfSelectedItem == totalNumberOfPages)
```

Comment:
Could these two `if` statements be collapsed to just `pageNoOfSelectedItem >= totalNumberOfPages`

Reply:
👍